src/StartServerPlugin.js

```diff
@@ -56,20 +58,23 @@ export default class StartServerPlugin {
 
   _getScript(compilation) {
     const {entryName} = this.options;
-    const map = compilation.entrypoints;
-    const entry = map.get ? map.get(entryName) : map[entryName];
+    const entrypoints = compilation.entrypoints;
+    const entry = entrypoints.get ? entrypoints.get(entryName) : entrypoints[entryName];
     if (!entry) {
+      console.error('Empty compilation.entrypoints');
       console.log(compilation);
       throw new Error(
-        `Requested entry "${entryName}" does not exist, try one of: ${(map.keys
-          ? map.keys()
-          : Object.keys(map)
+        `Requested entry "${entryName}" does not exist, try one of: ${(entrypoints.keys
+          ? entrypoints.keys()
+          : Object.keys(entrypoints)
         ).join(' ')}`
       );
     }
-    const entryScript = entry.chunks[0].files[0];
+    const entryScript = webpack.EntryPlugin ?
+      entry.runtimeChunk.files.values().next().value :
+      entry.chunks[0].files[0];
     if (!entryScript) {
-      console.error('Entry chunk not outputted', entry.chunks[0]);
+      console.error('Entry chunk not outputted', entry.runtimeChunk);
```

Comment:
This will be null on webpack 4, right?

Reply:
Yes, true. Will fix that 😀

Reply:
Changed it to print the entire entry since it is not that much data to show.